R/guide-bins.R

```diff
@@ -212,7 +212,7 @@ GuideBins <- ggproto(
       key$.value <- 1 - key$.value
     }
 
-    params$title <- scale$make_title(params$title %|W|% scale$name %|W|% title)
+    params$title <- scale$make_title(params$title, scale$name, title)
```

Comment:
Ideally, this would happen in the base `Guide` class, but axes don't have access to labels.